chart_app/lib/src/models/indicators.dart

```diff
@@ -320,21 +318,21 @@ class IndicatorsModel {
                   item.jawSeries!.entries,
                   epoch,
                   pipSize,
-                  offset: item.alligatorOptions.jawOffset,
+                  // offset: item.alligatorOptions.jawOffset,
                 ),
               if (item.teethSeries != null)
                 _getQuote(
                   item.teethSeries!.entries,
                   epoch,
                   pipSize,
-                  offset: item.alligatorOptions.teethOffset,
+                  // offset: item.alligatorOptions.teethOffset,
```

Comment:
here too

Reply:
It was commented while merging restoreDrawing branch into this branch and there were some error conflicts . It is added back now , Thanks